typescript/lib/networks/network_context.ts

```diff
@@ -0,0 +1,592 @@
+
+            const staticFileId: StaticFileID = <StaticFileID>(staticFileIdStr);
+
+            if (!staticFileFilepaths.has(staticFileId)) {
+                return err(new Error("No source filepath found for static file " + staticFileId + "; this is a bug in Kurtosis"));
+            }
+            const srcAbsFilepath: string = staticFileFilepaths[staticFileId];
+
+            const destAbsFilepath: string = path.join(this.enclaveDataVolMountpoint, destFilepathRelativeToEnclaveVolRoot);
+            
+            const promiseStatDestAbsFilepath: Promise<ResultAsync<fs.Stats, Error>> = new Promise((resolve, _unusedReject) => {
+                fs.stat(destAbsFilepath, (error: Error, response: fs.Stats) => {
+                    if (error === null) {
+                        resolve(okAsync(response));
+                    } else {
+                        resolve(errAsync(error));
+                    }
+                })
+            });
+            const resultStatDestAbsFilepath: Result<fs.Stats, Error> = await promiseStatDestAbsFilepath;
+            if (!resultStatDestAbsFilepath.isOk()) {
+                return err(new Error("The Kurtosis API asked us to copy static file " + staticFileId + " to path " + destFilepathRelativeToEnclaveVolRoot + 
+                " in the enclave volume which means that an empty file should exist there, " + "but no file exists at that path - this is a bug in Kurtosis!"));
+            }
+    
+            const promiseOpenSrcFp: Promise<ResultAsync<number, Error>> = new Promise((resolve, _unusedReject) => {
+                fs.open(srcAbsFilepath, 'r', (error: Error, response: number) => {
+                    if (error === null) {
+                        resolve(okAsync(response));
+                    } else {
+                        resolve(errAsync(error));
+                    }
+                })
+            });
+            const resultOpenSrcFp: Result<number, Error> = await promiseOpenSrcFp;
+            if (!resultOpenSrcFp.isOk()) {
+                return err(resultOpenSrcFp.error);
+            }
+            let srcFp: number;
+            
+            try {
+
+                srcFp = resultOpenSrcFp.value;
+                const promiseOpenDestFp: Promise<ResultAsync<number, Error>> = new Promise((resolve, _unusedReject) => {
+                    fs.open(destAbsFilepath, 'w', (error: Error, response: number) => {
+                        if (error === null) {
+                            resolve(okAsync(response));
+                        } else {
+                            resolve(errAsync(error));
+                        }
+                    })
+                });
+                const resultOpenDestFp: Result<number, Error> = await promiseOpenDestFp;
+                if (!resultOpenDestFp.isOk()) {
+                    return err(resultOpenDestFp.error);
+                }
+                 let destFp: number;
```

Comment:
Same thing here - should be `const destFp: number = resultOpenDestFp.value`

Also, the indentation here doesn't match the brace directly above it